libkbfs/folder_branch_ops.go

```diff
@@ -1631,24 +1625,58 @@ func (fbo *folderBranchOps) readyBlockMultiple(ctx context.Context,
 
 func (fbo *folderBranchOps) unembedBlockChanges(
 	ctx context.Context, bps *blockPutState, md *RootMetadata,
-	changes *BlockChanges, uid keybase1.UID) (err error) {
+	changes *BlockChanges, uid keybase1.UID) error {
 	buf, err := fbo.config.Codec().Encode(changes)
 	if err != nil {
-		return
+		return err
 	}
-	block := NewFileBlock().(*FileBlock)
-	block.Contents = buf
-	info, _, err := fbo.readyBlockMultiple(
-		ctx, md.ReadOnly(), block, uid, bps)
-	if err != nil {
-		return
+
```

Comment:
The other cost to that method is a few lines of duplicated code within the `if` statement (readying the block, setting block changes, adding ref bytes, etc).  I'll see if I can factor it out, but that's the reason I did it this way originally.